### DISCUSSION TITLE
Should anchors be centered in the middle of grid cells?

### DISCUSSION THREAD

**Discussion by seankao247** [Project-MONAI/MONAI#8268] [Q&A]
Hi everyone,
I’m currently working on resolving[ issue #7951](https://github.com/Project-MONAI/MONAI/issues/7951) regarding the anchor centering problem.

Centering anchors within grid cells has several benefits, including improved alignment with object regions, better feature matching during detection, and overall enhanced accuracy for downstream tasks like object localization. However, this approach is not currently implemented in MONAI.

Given these potential advantages, should we consider optimizing the anchor placement to achieve proper centering? I’d greatly appreciate your thoughts and suggestions on this matter.
Thank you!

**Comment by KumoLiu** [comment]
Hi @seankao247, welcome to the community, and thank you for your contribution! Please refer to our [Contribution Guide](https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md) for guidelines.

To maintain backward compatibility, please consider adding an argument that supports both options.
Thanks.
